test/integration/game_authentication_checks_test.rb

```diff
@@ -0,0 +1,74 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.